### PR TITLE
Ge 17

### PR DESCRIPTION
**Pull request by bgschiller**
Addresses #17

### CODE REVIEW

File: src/store/modules/models/libconfig.js [hunk 1]
```diff
@@ -58,6 +58,45 @@ const map = {
     ],
     init: factory.ThermalZone,
   },
+  window_definitions: {
```
Comment (gh-23378) [inline src/store/modules/models/libconfig.js]: Are window and daylighting control instances meant to be included in this file? we may want to remove those unless they are being used somewhere

File: src/components/Grid/Grid.vue [hunk 2]
```diff
@@ -37,9 +45,16 @@ export default {
...
+      drawWindow: drawWindow()
+        .xScale(v => this.rwuToGrid(v, 'x'))
+        .yScale(v => this.rwuToGrid(v, 'y')),
+      drawDC: drawDaylightingControl()
+        .xScale(v => this.rwuToGrid(v, 'x'))
+        .yScale(v => this.rwuToGrid(v, 'y')),
     };
   },
   mounted() {
+    window.theGrid = this;
```
Comment (gh-23378) [inline src/components/Grid/Grid.vue]: do we want to keep this?

File: src/components/Grid/Grid.vue [hunk 3]
```diff
@@ -144,10 +164,14 @@ export default {
 
     currentMode() { this.drawPolygons(); },
     polygons() { this.drawPolygons(); },
-
+    windowDefs: {
+      handler() { console.log('windowDefs was updated'); this.drawPolygons(); },
```
Comment (gh-23378) [inline src/components/Grid/Grid.vue]: did this actually need to be a deep watcher, or was the problem just caused by the ThermalZone typo?

File: src/components/Toolbar.vue [hunk 4]
```diff
@@ -84,7 +98,12 @@ import SaveAsModal from './Modals/SaveAsModal.vue';
 export default {
   name: 'toolbar',
   data() {
+    window.theToolbar = this;
```
Comment (gh-23378) [inline src/components/Toolbar.vue]: Are we using this?

File: src/store/modules/models/libconfig.js (hunk 1 above)
Comment (bgschiller) [reply src/store/modules/models/libconfig.js]: I don't think I understand. Where else would they be? Isn't this how we define something that will live in the library?

**Review by gh-23378 — APPROVED** [review]
sweet! thanks!

**Pull request merged**